proc-macros/src/discriminant.rs

```diff
@@ -59,16 +59,19 @@ pub fn derive_discriminant_impl(input_item: TokenStream) -> syn::Result<TokenStr
 			let a_span = a.span();
 			a.path
 				.is_ident("discriminant_attr")
-				.then(|| match syn::parse2::<ParenthesizedTokens>(a.tokens) {
-					Ok(ParenthesizedTokens { tokens, .. }) => {
-						let attr: Attribute = syn::parse_quote! {
-							#[#tokens]
-						};
-						Some(attr)
-					}
-					Err(e) => {
-						attr_errs.push(syn::Error::new(a_span, e));
-						None
+				.then(|| {
+					let _ = &a;
```

Comment:
Is this necessary? @TrueDoctor

Reply:
I mean that line seem to do absolutely nothing, did you try without it?

Reply:
I did not try without it since I don't know precisely what to test for in case of breakages. I was hoping you might be able to tell if it'd theoretically be a problem. See: https://doc.rust-lang.org/edition-guide/rust-2021/disjoint-capture-in-closures.html